docs/guides/main.rst

```diff
@@ -4,6 +4,8 @@ Getting Started
 Welcome to `beets`_! This guide will help you begin using it to make your music
 collection better.
 
+New to the CLI? Check out our :doc:`beginner friendly</guides/beginnerCLI>` getting started guide for Windows!
```

Comment:
I believe it would be better to name-space the documentation and thus this link to it:

`/guides/beginnerCLI` ➡️ `/guides/beginner-cli-windows`